lib/logger/lib/logger.ex

```diff
@@ -885,8 +891,8 @@ defmodule Logger do
         Logger.#{level}("#{message}")
 
```

Comment:
We need to improve the generated docs to show an example of structured logging too. :)

Reply:
Ok, so I will prepare that as well.

Reply:
Added docs for reports as well.